jwst/tweakreg/tweakreg_step.py

```diff
@@ -52,7 +68,7 @@ class TweakRegStep(Step):
         nclip = integer(min=0, default=3) # Number of clipping iterations in fit
         sigma = float(min=0.0, default=3.0) # Clipping limit in sigma units
         align_to_gaia = boolean(default=False)  # Align to GAIA catalog
```

Comment:
The more I look at all of these arguments I wonder if we need large-scale change in their names (similar to what some of us were discussing in a meeting yesterday). If we're now allowing `gaia_catalog` to point to any user-supplied catalog, then it just doesn't seem to make sense to me to keep the name "gaia" in all the other arguments that control how that catalog gets used. Why should the user specify ``align_to_gaia=True`` when they're aligning to their own catalog (not Gaia). Similarly for `min_gaia` and `save_gaia_catalog` (the latter of which probably isn't even needed or useful when the user is supplying the catalog to begin with - why save it again for them?).

The existing changes in the PR certainly make it possible, in the short term, for a user to specify their own catalog, but in long term I think it needs more thought and rework. What new param names like `align_to_refcat`, `ref_catalog`, `min_refcat`?

@stsci-hack @nden @mairanteodoro any thoughts?

Reply:
Yes, that is what @mairanteodoro and I were proposing: to get rid of `gaia` in parameter names. It is just that the work on exposing hidden parameters for the alignment of all images *at once* is work that @mairanteodoro is doing. Once that work is done, we should have a separate PR that deprecates the use of old parameters and introduces new parameters. I do not think this should be done here, unless you insist.

I would actually suggest that we get rig of `align_to_gaia(or anything)` altogether and simply specifying a reference catalog (user-supplied or `'GAIADR2'`) in `abs_refcat` would signal to run the absolute alignment.